repos/os/src/server/event_filter/button_scroll_source.h

```diff
@@ -182,8 +182,8 @@ class Event_filter::Button_scroll_source : public Source, Source::Filter
 					 * execute both conditions regardless of the result of the
 					 * first call of 'handle_activation'.
 					 */
-					if (_vertical_wheel  .handle_deactivation(event)
-					  | _horizontal_wheel.handle_deactivation(event)) {
+					if (_vertical_wheel  .handle_deactivation(event) ||
+					    _horizontal_wheel.handle_deactivation(event)) {
```

Comment:
The bitwise OR is indeed intended here. Please consider the comment above the `if`.